### ISSUE TITLE
add explanation next to `#[cfg(TODO)]`s in code

### ISSUE THREAD

**Issue by Lotterleben** [ferrous-systems/embedded-trainings-2020#137]
so that participants wondering why they can't instantiate a thing get a hint right where the problem is (I think it's mentioned in the book already)

**Comment by japaric** [comment]
I think it may be best to use block comments (`/* .. */`) which is more familiar to C and non-Rust programmers.

**Comment by Lotterleben** [comment]
Agreed. maybe we can sneak in a "🔎 in Rust projects, you may encounter another block comment notation: `#[cfg(TODO)]` effectively comments out the following element, which could be anything from a struct member to a module" in the materials?
